Python/RSA_Key_Generation/RSA.py

```diff
@@ -0,0 +1,148 @@
+import sys
```

Comment:
We have the RSA script here as well. WHich i believe s from your other PR. I am assuming you made a branch for this PR from the branch you made for your RSA script.

@SmritiAgrawal04 Please close this PR, make a new branch from master. Add the camscanner script so that I can review it.